src/Toaster/index.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+import Portal from '../Portal';
+import { Position } from '../Position';
+import { default as Toast, Props as ToastProps } from '../Toast';
+
+export type ToasterPosition =
+  | typeof Position.TOP
+  | typeof Position.TOP_LEFT
+  | typeof Position.TOP_RIGHT
+  | typeof Position.BOTTOM
+  | typeof Position.BOTTOM_LEFT
+  | typeof Position.BOTTOM_RIGHT;
+
+export const ToasterDefaultPosition = Position.TOP;
+
+type ToastData = ToastProps & { key: string };
+
+interface Props {
+  container: HTMLDivElement;
+}
+
+interface State {
+  toasts: ToastData[];
+}
+
+interface ToasterInterface {
+  show(props: ToastProps): void;
+}
+
+export default class Toaster extends React.Component<Props, State> implements ToasterInterface {
+  public static async create(container = document.body): Promise<Toaster> {
+    const containerElement = document.createElement('div');
+    containerElement.className = 'class101-ui-toaster';
+    container.appendChild(containerElement);
+
+    const toasterRef = React.createRef<Toaster>();
+
+    await ReactDOM.render<Props>(<Toaster container={containerElement} ref={toasterRef} />, containerElement);
```

Comment:
이거 ReactDOM.render 결과값이 Promise야?

Reply:
최신스펙은 그러합니당

> ReactDOM.render() currently returns a reference to the root ReactComponent instance. However, using this return value is legacy and should be avoided because future versions of React may render components asynchronously in some cases. If you need a reference to the root ReactComponent instance, the preferred solution is to attach a callback ref to the root element.